docs/art/retransmit_stage.bob

```diff
@@ -0,0 +1,45 @@
+                                              +------------+
+                                              |  Gossip    |
+                                              |  Service   |
+                                              |            |
+                                              +------------+
+                                                     |Repair peers
+                                                     |
+                          +------------------------------------------------------------------------------+
+                          |                          |                                                   |
+                          |  Retransmit Stage        |                                                   |
+                          |                          |                                                   |
+                          |  +---------------------------------------------+                             |
+                          |  |Window Service         |                     |                             |          +---------+
+                          |  |                       |                     |          +---------------+  |          |         |
+           Packets        |  +---------+  +--------+ |                     |  Shreds  |               |  | Entries  | Replay  |
+          +----------------->+Deshred  +->+Filter  +--------------------------------->+ Blockstore    +------------>+ Stage   |
```

Comment:
We have a component called Filter?

Reply:
I see, a function?  Since we only have blocks to work with, let's make them all microservice names.  So "Deshred" can be "Deshredder", and I'd guess "Filter" will have to be left out for now.

Reply:
Yea, it was to show shreds getting filtered by leader pubkey and there are some other checks on the slot and shred version.

Reply:
I changed to just Deshredder